docs/cli/cmdline/cloud-drift-show.md

```diff
@@ -0,0 +1,22 @@
+---
+title: terramate cloud drift show - Command
```

Comment:
maybe not worth saying but `terramate cloud` is not the correct command. Should not we match the actual command here? `terramate experimental cloud drift show`.

Reply:
I agree, but I was following the precedent set by the `cloud info` and `cloud login` commands which do not have `experimental` in the title.

Reply:
hmm true. Well, it's up to you guys. @soerenmartius

Reply:
We don't show `experimental` in the title :) This would change always when we move out of the experimental stage and folks don't copy the command from the title but from the examples :) hope that makes sense